app/models/comment.rb

```diff
@@ -39,6 +39,8 @@ def create_subscription
   def notify(action:, actor:, recipients:)
     case action.to_s
     when 'create'
+      create_notifications(action: :mention, actor: actor, recipients: recipients)
```

Comment:
Will this create duplicated notifications since it isn't excluded like on line: 52 for mentions?

Reply:
Not currently since line 52 checks for users with subscriptions and we don't have that implemented yet.

Reply:
Since we don't have a check in https://github.com/secroots/dradis-pro/blob/develop/app/models/concerns/notifiable.rb#L8, wouldn't this raise an error if `recipients` is `nil`?

Reply:
Looks like I'm mistaken and the comment builds a subscription on creation. I added the recipients to the line with exclusions to prevent doubling of the notifications.